clang/lib/StaticAnalyzer/Checkers/ErrnoModeling.h

```diff
@@ -13,6 +13,7 @@
 #ifndef LLVM_CLANG_LIB_STATICANALYZER_CHECKERS_ERRNOMODELING_H
 #define LLVM_CLANG_LIB_STATICANALYZER_CHECKERS_ERRNOMODELING_H
 
+#include "clang/Analysis/CFG.h"
```

Comment:
Actually, if we only declare an API that uses a class as a by-value parameter, that doesn't need the definition of the class.